content/en/apps/guides/tasks/task-schema-parameters.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Understanding the parameters in the Task Schema"
+linkTitle: Task Schema Parameters
+description: >
+  Understanding the data which is passed into Task interfaces
+relatedContent: >
+  apps/reference/tasks
+  apps/guides/data/hydration
+  core/overview/db-schema#contacts-persons-and-places
+  core/overview/db-schema#reports
+  apps/tutorials/tasks-1
+
+---
+
+{{% pageinfo %}}
+This guide explains the parameters available in the Task Schema and important constraints governing the design of tasks.
+
+- Useful knowledge if you are stuck writing your first `appliesIf` predicate
+- Understanding the data which is available in the task system and important constraints
+- Understand the special significance of the appliesTo attribute
+{{% /pageinfo %}}
+
+Let's synthesize some knowledge about CHT applications to help clarify what is happening within the task system:
+
+. All contacts in CHT applications are organised into hierarchies. For more information, read the [Contact and User Management Tutorial]({{< ref "apps/tutorials/contact-and-users-1" >}}) or [schema for contact documents](< ref "core/overview/db-schema#contacts-persons-and-places" >).
+2. All reports in the system are linked to one (and only one) contact. For more information, read [App Forms Tutorial]({{< ref "apps/tutorials/app-forms" >}}) or the [schema for report documents](< ref "core/overview/db-schema#reports" >).
+3. Documents are stored [minified]({{< ref "apps/guides/data/hydration" >}}) (not hydrated). All data that is passed into the tasks system is minified.
+4. Settings which control the documents which are available on the user's device are an important considerations to remember (eg [replication depth]("apps/guides/performance/replication#depth") or [purging]({{{< ref "apps/guides/performance/purging" >}})) since both tasks can only process docs which are present on the device.
```

Comment:
```suggestion
1. All contacts in CHT applications are organised into hierarchies. For more information, read the [Contact and User Management Tutorial]({{< ref "apps/tutorials/contact-and-users-1" >}}) or [schema for contact documents]({{< ref "core/overview/db-schema#contacts-persons-and-places" >}}).

2. All reports in the system are linked to one (and only one) contact. For more information, read [App Forms Tutorial]({{< ref "apps/tutorials/app-forms" >}}) or the [schema for report documents]({{< ref "core/overview/db-schema#reports" >}}).

3. Documents are stored [minified]({{< ref "apps/guides/data/hydration" >}}) (not hydrated). All data that is passed into the tasks system is minified.

4. Settings which control the documents which are available on the user's device are an important considerations to remember (eg [replication depth]({{< ref "apps/guides/performance/replication#depth" >}}) or [purging]({{< ref "apps/guides/performance/purging" >}})) since both tasks can only process docs which are present on the device.
```